src/buildHTML.js

```diff
@@ -311,43 +308,33 @@ groupTypes.supsub = function(group, options) {
         return groupTypes[group.value.base.type](group, options);
     }
 
-    const base = buildGroup(group.value.base, options.reset());
-    let supmid;
-    let submid;
-    let sup;
-    let sub;
+    const base = buildGroup(group.value.base, options);
+    let supm;
+    let subm;
```

Comment:
So, in the old code, `sub` and `sup` were not decorated with the classes necessary to change size, whereas `supmid` and `submid` were so wrapped.

In the new code, we don't need both versions, because we create the wrapped versions in one step. These are more like the `mid` versions, so I kept a suffix.